airgun/widgets.py

```diff
@@ -743,6 +743,35 @@ def select(self, item):
         self.browser.element(self.ITEM_LOCATOR.format(item), parent=self).click()
 
 
+class Pf4SvgActionsDropdown(Pf4ActionsDropdown):
+    """PF4 version of actions dropdown with svg image
+
+    Example html representation::
+
+        <div class="pf-c-dropdown pf-m-expanded" data-ouia-component-type="PF4/Dropdown">
+          <button class="pf-c-dropdown__toggle pf-m-plain" type="button">
+            <svg role="img">
+              <path d="...">
+              </path>
+            </svg>
+          </button>
+          <ul class="pf-c-dropdown__menu pf-m-align-right" role="menu">
+            <li role="none">
+              <button type="button" role="menuitem">Delete
+              </button>
+            </li>
+          </ul>
+        </div>
+
+    """
+
+    dropdown = Text('.//button[contains(@class,"pf-c-dropdown__toggle")]')
+    ITEM_LOCATOR = ".//ul/li/button[@role='menuitem' and contains(normalize-space(.), '{}')]"
+
+    def fill(self, item):
+        raise NotImplementedError
```

Comment:
for now I dont feel it might not be needed to write new widget, can you please try this https://github.com/SatelliteQE/airgun/blob/master/airgun/views/acs.py#L362 ?